web/client/epics/identify.js

```diff
@@ -174,7 +178,10 @@ module.exports = {
             const {disableAlwaysOn = false} = (store.getState()).mapInfo;
             return disableAlwaysOn || !stopFeatureInfo(store.getState() || {});
         })
-            .map(({point, layer}) => featureInfoClick(point, layer)),
+            .switchMap(({point, layer}) => Rx.Observable.of(featureInfoClick(point, layer))
+                .merge(Rx.Observable.of(addPopup(uuid(), { component: PopupViewer, maxWidth: 600, position: {  coordinates: point ? point.rawPos : []}}))
```

Comment:
We would prefer to not store component in the state via actions.
@mbarto suggested
> I would use strings, and eventually store string -> component association in a different place, outside of the store